skimage/transform/tests/test_warps.py

```diff
@@ -374,14 +378,17 @@ def test_resize_dtype():
     assert resize(x_f32, (10, 10), preserve_range=True).dtype == x_f32.dtype
 
 
-def test_swirl():
-    image = img_as_float(checkerboard())
+@testing.parametrize('dtype', [np.float32, np.float64])
```

Comment:
```suggestion
@testing.parametrize('dtype', [np.float16, np.float32, np.float64])
```
Why not include float16, knowing that it's expected to be promoted to float32?